src/qri/dataset.ts

```diff
@@ -92,6 +92,27 @@ export type ComponentName =
  | 'viz'
  | 'stats'
 
+// rankedComponentNames is a ranked list ranging from the components we deem
+// it most important that user see first, to the least important
+const rankedComponentNames = ['body', 'readme', 'transform', 'viz', 'meta', 'stats', 'commit', 'structure']
```

Comment:
@chriswhong what do you think of this ranking? Please let me know what you think the order should be if your thoughts differ!

Reply:
I don't have strong opinions on this.  It's fine for now, we can tweak it later if necessary.